health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserAccessMgmt.js

```
@@ -0,0 +1,41 @@
+    return (
+        <div>
+        <Card>
+        <Header styles={{ fontSize: "32px" }}>{t("USER_ACCESS_MGMT")}</Header>
+        </Card>
+        {rolesArray.map((role) => {
+            // Step 2: Call the function with the provided data
+
+            return(
+              <UserAccessMgmtTableWrapper
+              role={role}
+              />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)